ubuntu.py

```diff
@@ -171,7 +172,7 @@
         resp_time_curve_fit_parameters = np.zeros((12, 4))
         conv_ref_time_curve_fit_parameters = np.zeros((12, 5))
         for month in range(1, 13):
```

Comment:
even line-174 can go through the calendar module. The months can only go from 1 to 12, not 13.

Reply:
All of the arithmetic on month variable can be replaced with functions from calendar module.